src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
```

Comment:
Please move your numbers (40, 7, 50, 3, 20) in constant, like:
const `COST_PER_DAY = 40`, `BIG_DISCOUNT = 50`, ...